src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal page links. If these links are meant to navigate to 'index.html', consider if this is the correct behavior for each navigation item.